README.md

```diff
@@ -312,10 +335,22 @@ class ApplicationController < ActionController::Base
 end
 ```
 
+#### 2.b.2. Routes
+
+```ruby
+Rails.application.routes.draw do
+  # ...
+  resources :users
+  resource :user_session
```

Comment:
IMO we should point out that singleton routes are a good idea for Authlogic: you never pass session ids in a url, and AFAIK we'd only ever be dealing with 1 type of session at time? This also makes the logout url pretty easy to deal with: `link_to "Logout", user_session_url, method: :delete`